### PR TITLE
Feature/iteration 1 michannstoner

### PR DESCRIPTION
**Pull request by michannstoner**
Requesting code review for new functionality: Being able to view the saved posters section, create poster form, and return back to the main page from either section.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,6 +3,23 @@ var randomPosterButton = document.querySelector('.show-random');
 var mainPosterImage = document.querySelector('.poster-img');
 var posterTitle = document.querySelector('.poster-title');
 var posterQuote = document.querySelector('.poster-quote');
+
```
Comment (Meekb) [inline src/main.js:6]: Thank you for sectioning out the variable lists by type! Much easier to read.

File: src/main.js [hunk 2]
```diff
@@ -106,14 +123,43 @@ var currentPoster;
 
 // event listeners go here 👇
 randomPosterButton.addEventListener('click', makeRandomPoster)
+
```
Comment (Meekb) [inline src/main.js:126]: helper functions! great job.

**Review by Meekb — APPROVED** [review]

File: src/main.js [hunk 3]
```diff
@@ -106,14 +123,43 @@ var currentPoster;
...
+function hideMainPage(page) {
+  page.classList.remove('hidden');
+  mainPosterSection.classList.add('hidden');
+};
+
+function takeBackToMain(page) {
+  page.classList.add('hidden');
+  mainPosterSection.classList.remove('hidden');
+};
```
Comment (MarcelineBall) [inline src/main.js:163]: Nice refactor to make the takeBackToMain more versatile

File: src/main.js (hunk 1 above)
Comment (MarcelineBall) [reply src/main.js:6]: Great for programmer empathy

**Review by MarcelineBall — APPROVED** [review]
Code is functional, and was made more clean. Great job!

**Pull request merged**